### PR TITLE
tmux-xpanes: init at 4.1.0

### PR DESCRIPTION
**Pull request by monsieurp**
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by monsieurp** [comment]
:heavy_check_mark:

File: pkgs/tools/misc/tmux-xpanes/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, openssl, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "tmux-xpanes";
+  version = "4.1.0";
+
+  src = fetchFromGitHub {
+    owner = "greymd";
+    repo = "${pname}";
+    rev = "v${version}";
```
Comment (teto) [inline pkgs/tools/misc/tmux-xpanes/default.nix]: no need to quote: repo = pname;

Comment (monsieurp) [reply pkgs/tools/misc/tmux-xpanes/default.nix]: thanks, I've also remove the quotes across the other variabels. Please have a look and let me know.

Comment (monsieurp) [reply pkgs/tools/misc/tmux-xpanes/default.nix]: fixed!

**Comment by monsieurp** [comment]
:heavy_check_mark:

**Comment by monsieurp** [comment]
@GrahamcOfBorg build tmux-xpanes

**Comment by monsieurp** [comment]
@grahamc: Hi! Could you take a look at this PR if you get round to it? Thanks! :)

**Review by gazally — COMMENTED** [review]
I built this and tried out the ping example from the homepage and everything worked.

File: pkgs/tools/misc/tmux-xpanes/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    install -m 755 -d $out/bin/
+    install -m 755 -d $out/share/man/man1/
+
+    # Perform installation.
+    install -m 755 bin/* $out/bin/
+    install -m 644 man/*.1 $out/share/man/man1/
+  '';
+
+  meta = with stdenv.lib; {
+    description = "tmux-based terminal divider";
```
Comment (gazally) [inline pkgs/tools/misc/tmux-xpanes/default.nix]:
```suggestion
    description = "Awesome tmux-based terminal divider";
```
This is what they call it on their homepage, and it fits the [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md) rule better.

**Comment by ryantm** [comment]
@GrahamcOfBorg build tmux-xpanes

**Pull request merged**